nvflare/edge/device/android/sdk/TrainerRegistry.kt

```diff
@@ -0,0 +1,72 @@
+package com.nvidia.nvflare.sdk
+
+import android.util.Log
+import com.nvidia.nvflare.sdk.training.Trainer
+import com.nvidia.nvflare.sdk.models.TrainingConfig
+
+/**
+ * Registry for trainer implementations.
+ * Follows the Python pattern for dynamic component resolution.
+ */
+object TrainerRegistry {
+    private val TAG = "TrainerRegistry"
+    private val trainerFactories = mutableMapOf<String, TrainerFactory>()
+
+    /**
+     * Factory interface for creating trainer instances.
+     */
+    interface TrainerFactory {
```

Comment:
Now we have only one Trainer ETTrainer, maybe no need for this TrainerRegistry.
ETTrainer should be applicable for all different ET models